### ISSUE TITLE
Model files missing in pretrained_model folder

### ISSUE THREAD

**Issue by charlzizz** [PetrochukM/Simple-QA-EMNLP-2018#1]
Hi.
Looks like we are unable to run through the end-to-end pipeline currently due to missing model checkpoints in pretrained_model folder.
`../../pretrained_models/relation_classifier.02_02_07:59:28/1746.pt`
`../../pretrained_models/relation_classifier.02_02_13:31:11/189.pt`
`../../pretrained_models/relation_classifier.02_02_13:23:02/223.pt`

**Comment by PetrochukM** [comment]
Yes, those models are not available, sorry everyone! I do not have them anymore.

**Comment by PetrochukM** [comment]
Please help! If anyone has trained new models in a reproduction effort, please upload your models via a pull request.

The hyperparameters and log files should be available to aid reproduction.

**Comment by HimaniChanchal** [comment]
While reproducing the results for Relation Classifier RNN model Accuracies are 0.0 and loss is 7.44. What is the issue?

**Comment by PetrochukM** [comment]
Please use this guide when describing your issue:
```
## 🐛 Bug

<!-- A clear and concise description of what the bug is. -->

## To Reproduce

Steps to reproduce the behavior:

1.
1.
1.

<!-- If you have a code sample, error messages, stack traces, please provide it here as well -->

## Expected behavior

<!-- A clear and concise description of what you expected to happen. -->
```

**Comment by HimaniChanchal** [comment]
Whenever I am training my Relation Classifier RNN model with provided code, I am getting a constant loss(7.44)  and 0.00 accuracy. My training files are correct and also model parameters are same as taken in tha paper. I am attaching my output as well as my parameters to the model.
![hyperparmeter](https://user-images.githubusercontent.com/27516315/64046793-66ac3d00-cb8a-11e9-8be6-2feed109c541.png)
![error](https://user-images.githubusercontent.com/27516315/64046801-6d3ab480-cb8a-11e9-8f24-2b21ffb77a42.png)

**Comment by PetrochukM** [comment]
Sorry, I won't be helping you. 

I don't have enough free time to help you with super broad questions with little to no specific details. 

Furthermore, this is the wrong thread to talk about your issue.
